easy_my_coop/security/ir.model.access.csv

```diff
@@ -5,7 +5,7 @@ access_product_template_easy_my_coop_user,access_product_template_easy_my_coop_u
 access_product_template_easy_my_coop_manager,access_product_template_easy_my_coop_manager,product.model_product_template,group_easy_my_coop_manager,1,1,1,1
 access_subscription_request_user,access_subscription_request_user.subscription,model_subscription_request,base.group_user,1,0,0,0
 access_subscription_request_easy_my_coop_user,access_subscription_request_easy_my_coop_user,model_subscription_request,group_easy_my_coop_user,1,1,1,0
-access_subscription_request_easy_my_coop_manager,access_subscription_request_easy_my_coop_manager,model_subscription_request,group_easy_my_coop_manager,1,1,1,1
+access_subscription_request_easy_my_coop_manager,access_subscription_request_easy_my_coop_manager,model_subscription_request,group_easy_my_coop_manager,1,1,1,0
```

Comment:
@coccinouille (my bad for not raising the issue in the review but ) do not forget to launch the tests before submitting a PR (it will be check by the CI in the future).

Reply:
@robinkeunen
Noted for launching the test before PR submission.
However I did not understand the rest of your comment. 
Which issue are you refering to ? I don't understand if you're expecting some code fix from me or not.

Reply:
Some comments were stil pending, sorry. I fixed it here 
https://github.com/coopiteasy/vertical-cooperative/commit/71f11e278eea7a93fcd8c621a879cc0e619c498a , thank you :-)